pkg/auth/rhsso_authz_handler.go

```diff
@@ -133,22 +132,8 @@ func (a *AuthzHandler) hasSubscriptionAccess(clusterId string, action string, pa
 		if err != nil {
 			return handleOwnershipQueryError(err)
 		}
-
-		cacheKey := fmt.Sprintf("%s_%s_%s_%s", payload.Username, payload.Organization, cluster.AmsSubscriptionID, action)
-		if cacheData, existInCache := a.client.Cache.Get(cacheKey); existInCache {
-			var ok bool
-			isAllowed, ok = cacheData.(bool)
-			if !ok {
-				return false, fmt.Errorf(
-					"error while retrieving cluster edit role from cache for %s",
-					cluster.AmsSubscriptionID.String())
-			}
-			return isAllowed, nil
-		}
+		// TODO: check if caching is required
```

Comment:
is there a ticket number for it? might be worth noting.

Reply:
I'll remove the todo, it's seems caching shouldn't be required for AccessReview: https://github.com/openshift/assisted-service/pull/3729#issuecomment-1116119258